language-guides/_includes/driver-warning.adoc

```diff
@@ -0,0 +1,10 @@
+
+[NOTE]
+====
+From Neo4j version 4.0 and onwards, the default encryption setting is *off* by default and Neo4j will no longer generate self-signed certificates when none is provided.
+When connecting to a Neo4j server with encryption enabled, you must use the `neo4j+s` scheme in your connection string for a trusted certificate or `neo4j+ssc` for a self-signed certificate.
```

Comment:
Do we need to say from which driver versions those schemes are supported? At least say 4.0 drivers? 
It's a bit confusing with db version vs. driver version?